### DISCUSSION TITLE
I couldn't deploy my first contract to ZkSync testnet.

### DISCUSSION THREAD

**Discussion by karan2004sahu** [Cyfrin/foundry-full-course-cu#2262] [Q&A] [answered]
zkSync is modifying .workspaces/Basic - 1/artifacts/contracts/simplestorage.sol/SimpleStorage.json
This error warning is showing, and I don't know what to do further to resolve this problem. I am using the wallet as mentioned in the tutorial, and MetaMask is also popping up to ask for the signature to deploy the contract, but after this step, things don't work.
![image](https://github.com/user-attachments/assets/e5954ef2-d592-42b3-b305-4b40ec44ce0d)

**Comment by EngrPips** [comment]
Hello @karan2004sahu, `Deploy & Verify` is not working appropriately. Please use `Deploy` instead.

**Reply by karan2004sahu** [reply]
thanks.

**Comment by karan2004sahu** [comment, marked as answer]
it just not working properly , it will work as expected give it some time reload your window and repeat the steps and you will get the result, i learned it myself after 20min.
best of luck

**Reply by EngrPips** [reply]
Excellent. Thanks for dropping the solution here. You are the best.

**Comment by yijiale-1120** [comment]
zksync plugin cannot be deployed and verified, only deployed

**Reply by EngrPips** [reply]
If you are able yo deploy and test your contract that is sufficient. Don't sweat over verification.
